packages/core/README.md

```diff
@@ -42,6 +46,7 @@ Each extension can consist of the following modules:
 - `frontendElectron` is used in the electron env
 - `backend` is used in the node env and as well in the electron env if `backendElectron` is not provided
 - `backendElectron` is used in the electron env
+- `electronMain` is used in the electron-main env
```

Comment:
I'm not sure this statement offers much value, to an application or extension developer what is an `electron-main` environment, and how is it different than the standard `electron` environment? In addition, I do not believe `electronMain` is for an "electron-main env" but rather an extension point to provide customizations to the `electron-main` process (https://github.com/eclipse-theia/theia/pull/7964).

Reply:
`is used in the electron-main process` could do the trick?